### DISCUSSION TITLE
Get-PnPListItem returns nothing

### DISCUSSION THREAD

**Discussion by hf740** [pnp/powershell#3571] [Q&A]
I'm trying to get all the items in a simple list. The list exists. I use Powershell 7.

`
# Parameters
$siteUrl      = "https://bah"
$clientId     = "6e**************************651******7e" 
$clientSecret = "******************83Hz0***************"
$listTitle     = "MyTitle"

Connect-PnPOnline -Url $siteUrl -ClientId $clientId -ClientSecret $clientSecret

$lists = Get-PnPList -Identity $listTitle
$lists | Format-Table  # OK!

$fields = Get-PnPField -List $listTitle
$fields | Format-Table # OK!

$items = Get-PnPListItem -List $listTitle
$Items # NO DATA!
Disconnect-PnPOnline`

The problem is that I cannot get any rows from Get-PnPListItem, but I can enumerate the lists, and enumerate the fields of the list in questions. But I can't get any data.

I've searched the web up and down, tried a lot of different approaches, but no use. What am I doing wrong?

**Comment by mprzenzak** [comment]
Use command `Get-PnPAzureADAppSitePermission -AppIdentity "YourAppId"` to check application role. It shold print "Role: Write/Read/FullControl". If you got "Roles: {none}", then it mean that the app does not have permissions.
